src/tseda/datastore.py

```diff
@@ -147,13 +147,13 @@ def tooltip(self):
             ),
         )
 
-    def sample_sets(self):
+    def sample_sets(self, only_selected=True):
```

Comment:
I would rename the parameter to `selected`; this is enough to imply that we only look at individuals tagged as selected.

Reply:
Tell you what, keep it as it is, else the next statement becomes confusing.